testsuite/forge/src/interface/node.rs

```diff
@@ -193,7 +199,7 @@ pub trait NodeExt: Node {
     }
 
     fn liveness_check(&self, seconds: u64) -> Result<()> {
```

Comment:
I think we added the healthcheck api to the rest client so we could use it here directly.